### PR TITLE
Fixed #26583 -- Added quiet option to staticfiles command

### PR DESCRIPTION
**Pull request by ryanhunter1836**

### CODE REVIEW

File: django/contrib/staticfiles/management/commands/collectstatic.py [hunk 1]
```diff
@@ -68,6 +68,11 @@ def add_arguments(self, parser):
             help="Don't ignore the common private glob-style patterns (defaults to 'CVS', '.*' and '*~').",
         )
 
+        parser.add_argument(
+            '--quiet', action='store_true', dest='quiet',
```
Comment (charettes) [inline django/contrib/staticfiles/management/commands/collectstatic.py:72]: Feels like `--ignore-conflicts` would be a better fit.

**Review by charettes — COMMENTED** [review]
I don't buy the rationale of the reporter of the ticket about the necessity of introducing a specialized option here. If they can adapt their script to pass and extra `--ignore-conflicts` flag they can certainly filter out lines with a `Found another file with the destination` prefix using `grep`.

We certainly want to warn about conflicts and provide a summary of the changes by default verbosity. Maybe we could turn the problem around and have the summary message include the number of skipped files by default

```
0 static files copied to '/home/django/djangoproject.com/static', 833 unmodified, 42 skipped due to conflict
```

And have the current verbose message only emitted if `verbosity >= 2`?

**Comment by felixxm** [comment]
I agree with Simon's [comment](https://github.com/django/django/pull/13683#pullrequestreview-530764272) a new option is not necessary.

**Comment by felixxm** [comment]
@ryanhunter1836  Do you have time to keep working on this?

**Comment by ryanhunter1836** [comment]
@felixxm I can keep working, I was just unsure of what to do next

**Comment by felixxm** [comment]
IMO, we should implement Simon's [suggestion](https://github.com/django/django/pull/13683#pullrequestreview-530764272):

> We certainly want to warn about conflicts and provide a summary of the changes by default verbosity. Maybe we could turn the > problem around and have the summary message include the number of skipped files by if any
> 
> ```
> 0 static files copied to '/home/django/djangoproject.com/static', 833 unmodified, 42 skipped due to conflict
> ```
> 
> And have the currently verbose message only emitted if `verbosity >= 2`?

**Comment by matthiask** [comment]
The same could be done for the `Deleted ...` message when passing `--clear` I think. `collectstatic` shouldn't emit one line per deleted file when using the default verbosity since you have to opt in to this behavior. 

(I know that this isn't exactly the same issue but I'm commenting here because it also concerns the very verbose output of `collectstatic`.)

**Comment by felixxm** [comment]
Closing due to inactivity.

**Pull request closed without merging**